backend/app/models/content.py

```
@@ -1,9 +1,12 @@
 import uuid
 from datetime import datetime
 
-from sqlalchemy import CheckConstraint, UniqueConstraint
-from sqlalchemy.dialects.postgresql import JSONB
-from sqlmodel import JSON, Column, Field, Relationship, SQLModel
+from sqlalchemy import CheckConstraint, UniqueConstraint, Column, String, Text, DateTime, Enum, Boolean, Integer, ForeignKey, Index
```

Ruff (0.12.2)

`sqlalchemy.String` imported but unused

Remove unused import

(F401)